src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.ts

```diff
@@ -26,21 +26,21 @@ export class OneHabitComponent implements OnInit {
     acquired: () => {
       this.daysCounter = this.habit.duration;
       this.showPhoto = false;
-      this.habitMark = 'empty';
+      this.habitMark = 'aquired';
     },
     done: () => {
       this.daysCounter = this.habit.workingDays
         ? this.habit.workingDays
         : this.habit.duration;
       this.showPhoto = false;
-      this.habitMark = 'mark';
+      this.habitMark = 'done';
     },
     undone: () => {
       this.daysCounter = this.habit.workingDays
         ? this.habit.workingDays
         : this.habit.duration;
       this.showPhoto = true;
-      this.habitMark = 'empty';
+      this.habitMark = 'undone';
```

Comment:
create enum in this file for habitMarkStatuses

Reply:
done